mars/deploy/oscar/ray.py

```diff
@@ -434,6 +439,35 @@ async def start(self):
             logging.basicConfig(
                 format=ray.ray_constants.LOGGER_FORMAT, level=logging.INFO
             )
+        execution_config = ExecutionConfig.from_config(
+            self._config, backend=self.backend
+        )
+        self.backend = execution_config.backend
+        if self.backend == "mars":
+            await self.start_oscar(
+                self._n_supervisor_process,
+                self._supervisor_mem,
+                self._worker_num,
+                self._worker_cpu,
+                self._worker_mem,
+            )
+        else:
+            execution_config.merge_from(
```

Comment:
Maybe it's better to specify the backend explicitly like `ray` or something others instead of `else` statement.
And we can raise an exception if user passes an unsupported backend.

Reply:
Good catch, we should validate the passed backend type, just fixed it.